Modules/Crm/Http/Controllers/ProjectsController.php

```
@@ -2,104 +2,60 @@
+            'filter_placeholder' => trans('filter_projects'),
+            'filter_method' => 'filter_projects',
+            'projects' => $projects,
+        ]);
     }
 
-    /**
-     * Display the specified resource.
-     */
-    public function show($id)
+    /** @legacy-file application/modules/projects/controllers/Projects.php:49 */
+    public function form(?int $id = null)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)